salttesting/jenkins.py

```diff
@@ -1752,7 +2226,10 @@ def main():
             time.sleep(1)
 
     if options.download_artifact:
-        download_artifacts(options)
+        if getattr(options, 'windows', False):
+            download_artifacts_smb(options)
```

Comment:
Why two different functions? It seems like it would make more sense to maintain `download_artifacts` and let that figure out what to do based on the options that are set.